fares-data-build-tool/create-data
# feat: CFD-473 Added my fares link

# Description

Provide a way for operators to navigate to my fares from the existing home page as this will become their default place to create/view/edit/export their fares. 



# Testing instructions

-   Pull to local and click "View and manage fares"
-  Also add a fare and check the "add another fare" button re-directs to "faretype"
# Type of change
